pkg/securitypolicy/securitypolicy_test.go

```diff
@@ -609,10 +609,10 @@ func Test_EnforceEnvironmentVariablePolicy_NarrowingMatches(t *testing.T) {
 		// create two additional containers that "share everything"
 		// except that they have different environment variables
 		testContainerOne := generateContainersContainer(r, 5)
-		testContainerTwo := testContainerOne
+		var testContainerTwo securityPolicyContainer = *testContainerOne
```

Comment:
same as above

Reply:
I agree. Test code should quality code too.